plugins/modules/oneview_appliance_time_and_locale_configuration.py

```diff
@@ -0,0 +1,117 @@
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_appliance_time_and_locale_configuration
+short_description: Manage OneView Appliance Locale and Time Configuration.
+description:
+    - Provides an interface to manage Appliance Locale and Time Configuration. It can only update it.
+version_added: "2.9.0"
+requirements:
+    - "python >= 3.4.2"
+    - "hpeOneView >= 6.0.0"
+author:
+    "Shanmugam M (@SHANDCRUZ)"
+options:
+    state:
+        description:
+            - Indicates the desired state for the Appliance Locale and Time Configuration.
+              C(present) will ensure data properties are compliant with OneView.
+        choices: ['present']
+        required: true
+        type: str
+    data:
+        description:
+            - List with the Appliance Locale and Time Configuration properties.
+        required: true
+        type: dict
+
+extends_documentation_fragment:
+    - hpe.oneview.oneview
+    - hpe.oneview.oneview.params
+'''
+
+EXAMPLES = '''
+- name: Ensure that the Appliance Locale and Time Configuration is present with locale 'en_US.UTF-8'
+  oneview_appliance_time_and_locale_configuration:
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 2600
+    state: present
+    data:
+      locale: 'en_US.UTF-8'
+'''
+
+RETURN = '''
+appliance_time_and_locale_configuration:
+    description: Has the facts about the OneView Appliance Locale and Time Configuration.
+    returned: On state 'present'. Can be null.
+    type: dict
+'''
+
+from ansible_collections.hpe.oneview.plugins.module_utils.oneview import OneViewModule, compare
+
+
+class ApplianceTimeAndLocaleConfigurationModule(OneViewModule):
+    MSG_CREATED = 'Appliance Locale and Time Configuration configured successfully.'
+    MSG_ALREADY_PRESENT = 'Appliance Locale and Time Configuration is already configured.'
+    RESOURCE_FACT_NAME = 'appliance_time_and_locale_configuration'
+
+    def __init__(self):
+        additional_arg_spec = dict(data=dict(required=True, type='dict'),
+                                   state=dict(
+                                       required=True,
+                                       choices=['present']))
+
+        super(ApplianceTimeAndLocaleConfigurationModule, self).__init__(additional_arg_spec=additional_arg_spec)
```

Comment:
super doesn't expect arguments for super() method in python3
https://stackoverflow.com/questions/10482953/python-extending-with-using-super-python-3-vs-python-2